examples/aggregator/example.toml

```diff
@@ -11,7 +11,8 @@ manifest = "examples/aggregator/oak_app_manifest.toml"
 out = "examples/aggregator/bin/aggregator.oak"
 
 [applications.rust.modules]
-module = { Cargo = { cargo_manifest = "examples/aggregator/module/rust/Cargo.toml" } }
+main_module = { Cargo = { cargo_manifest = "examples/aggregator/main_module/rust/Cargo.toml" } }
+handler_module = { Cargo = { cargo_manifest = "examples/aggregator/handler_module/rust/Cargo.toml" } }
```

Comment:
I don't think having the extra /rust/ in the path is particularly useful, can we just remove them?

Reply:
Done